src/WebJobs.Script/Diagnostics/Extensions/LoggerExtension.cs

```diff
@@ -280,19 +280,19 @@ public static void FunctionMetadataManagerLoadingFunctionsMetadata(this ILogger
             _functionMetadataManagerLoadingFunctionsMetadata(logger, null);
         }
 
-        public static void FunctionMetadataManagerFunctionsLoaded(this ILogger logger, int count)
+        public static void FunctionsLoadedByProvider(this ILogger logger, int count)
         {
-            _functionMetadataManagerFunctionsLoaded(logger, count, null);
+            _functionMetadataProviderFunctionsLoaded(logger, count, null);
         }
 
-        public static void FunctionMetadataProviderParsingFunctions(this ILogger logger)
+        public static void FunctionMetadataProviderParsingFunctions(this ILogger logger, string provider)
```

Comment:
As per the comment, this log method should also be renamed to ReadingFunctionMetadataFromProvider